spec/features/user_sees_dashboard_functionality_when_signed_in_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'As a logged in user' do
+  before :each do
+    visit root_path
+    stub_omniauth
+    click_link 'Sign in with Google'
+
+    expect(current_path).to eq(dashboard_path)
+    expect(page).to have_content('Happy')
+    expect(page).to have_content('Gilmore')
+  end
+
+  it "shows options for users via dropdown menus before they shake it up" do
+    expect(page).to have_content('Welcome, Happy Gilmore')
+    expect(page).to have_content('Logout')
+    select "1 Mile", from: :distance
+    select "$$$", from: :prices
+    click_button 'Shake It Up!'
+  end
+end
+# As a user
```

Comment:
The user story can be removed now. Also, once the restaurant show page is integrated we can test that the current_path is the restaurant/show after we click "shake it up".